ghcli-darwin.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+###################################
+# Installs github cli tool for Mac
+# Author: github.com/sosaeio
+# Date: Oct 28, 2023
+###################################
+
+if [ $(which gh | grep -o gh > /dev/null && echo 1 || echo 0) -eq 1 ]; then
```

Comment:
All you need here is:

`if ( which gh )`

That will pass if exit 0, fail otherwise.

If statements in Bash use `()` for subshell exec and exit code and `[]` for implicit `test` commands like `-d -s -z`.